web/src/vdom.ts

```diff
@@ -74,7 +99,12 @@ export function update_attrs($elem, new_attrs, old_attrs) {
     }
 }
 
-export function update(replace_content, find, new_dom, old_dom) {
+export function update(
+    replace_content: (arg0: string | undefined) => void,
+    find: () => JQuery<HTMLElement>,
+    new_dom: {opts: VDomOpts; tag_name: string},
+    old_dom?: {opts: VDomOpts} | undefined,
```

Comment:
```suggestion
    old_dom?: {opts: VDomOpts},
```
You can remove ` | undefined` here as it's unnecessary.